src/screens/DomainScreen/elements/Header/elements/DomainFollowerNumber.js

```diff
@@ -1,40 +1,41 @@
 import * as React from 'react';
-import {StyleSheet, Text, View} from 'react-native';
+import { StyleSheet, Text, View } from 'react-native'
 
-import {COLORS} from '../../../../../utils/theme';
-import {Gap} from '../../../../../components';
-import {fonts, normalize, normalizeFontSize} from '../../../../../utils/fonts';
-import {getSingularOrPluralText} from '../../../../../utils/string/StringUtils';
+import { COLORS } from '../../../../../utils/theme';
+import { Gap } from '../../../../../components';
+import { colors } from '../../../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You have imported both `COLORS` and `colors`. If you are replacing `COLORS` with `colors`, you should remove the import of `COLORS`.
```diff
- import { COLORS } from '../../../../../utils/theme';
  import { Gap } from '../../../../../components';
  import { colors } from '../../../../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->